### PR TITLE
Rebrand server arm download page wd 9427

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Check https://ubuntu-com-13668.demos.haus/download/server/arm against [copy doc](https://docs.google.com/document/d/1whVU3DvB9j5k6Ed3UvLZCEN6BU-lZcow4ZefqKQqokg/edit)
- Check against design screenshot below

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9427

## Screenshots (design review)

arm

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@carkod looks good, a few minor improvements:

- duplicate  hr here (see how it gets slightly darker)

image

- hide this hr on mobile 

image

- p-section spacing under the link
 
image

- could you please adjust the medium breakpoints on these sections to be indented 1 column, like so: 

image
(live version here https://ubuntu.com//desktop/organisations)

**Comment by juanruitina** [comment]
Only two comments:

- Please change the "Download 22.04.4 LTS (64k page size)" button to white (default). I just tweaked it in the copy doc and design.
- This is visual, but I see in the design some more spacing between the hero paragraphs ("... retaining the reliable and familiar Ubuntu experience.") and the separator below. I think it will look tidier if we add it.

**Comment by carkod** [comment]
> Only a couple of comments:
> 
> * Please change the "Download 22.04.4 LTS (64k page size)" button to white (default). I just tweaked it in the copy doc and design.
> * This is visual, but I see in the design some more spacing between the hero paragraphs ("... retaining the reliable and familiar Ubuntu experience.") and the separator below. I think it will look tidier if we add it.
> * For some reason this heading wraps on mobile:
> 
> Screenshot 2024-03-14 at 12 59 06
> 
> Feel free to change the label to UX+1 when you fix this.

Thanks. All fixed.

**Comment by lyubomir-popov** [comment]
- this now has too much space - 2 p-sections nested, we need 1 

image

- these need the same treatment as further up - bold headings and line above 1 column in:

image

- which will make them same as these above: 
- 
image

these are a bit inconsistent - I'd put lines above each link:
image

File: templates/download/server/arm.html [hunk 1]
```diff
@@ -5,124 +5,147 @@
...
+        <div class="col-6 col-medium-3 p-section">
+          <p>This ISO image is suited for servers with ample memory running memory-intensive applications.</p>
+          <hr />
+          <a href="https://cdimage.ubuntu.com/releases/{{ releases.lts.short_version }}/release/ubuntu-{{ releases.lts.full_version }}-live-server-arm64+largemem.iso" class="p-button" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'ARM server download', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">
             Download {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr> (64k page size)
           </a>
-        </p>
-        <p><a href="/server/docs/choosing-between-the-arm64-and-arm64-largemem-installer-options">Find out if 64k page size is for you&nbsp;&rsaquo;</a></p>
+          <p>
+            <a href="/server/docs/choosing-between-the-arm64-and-arm64-largemem-installer-options">Find out if 64k page size is for you&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/download/server/arm.html:55]: I am getting a 404 from this link

File: templates/download/server/arm.html [hunk 2]
```diff
@@ -5,124 +5,147 @@
...
+          <hr class="p-rule" />
+          <p><a href="/pro">Ubuntu Pro&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Helping hands</h3>
+        </div>
+        <div class="col-6 col-medium-3">
```
Comment (petesfrench) [inline templates/download/server/arm.html:133]: This section is missing the text 'If you get stuck, help is always at hand.'

File: templates/download/server/arm.html [hunk 3]
```diff
@@ -5,124 +5,147 @@
...
+      <p>Pair your ARM server deployment with enterprise-grade 24/7 support with <a href="/pro">Ubuntu Pro</a> to get the SLA-backed assurance that you are fully covered by our system and architecture experts &mdash; no matter what comes up.</p>
       <p>Built for cutting-edge hardware, from the HP Moonshot range to standard form-factor certified systems, Ubuntu and ARM Server provide truly compelling economics for cloud-scale deployment.</p>
+      <hr class="p-rule" />
       <p><a href="/server/contact-us?product=server-arm">Contact us about your ARM server plans&nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
 
-{% with first_item="_download_server_arm_guide", second_item="_download_enterprise_support", third_item="_download_helping_hands" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+<section class="p-section--deep">
```
Comment (petesfrench) [inline templates/download/server/arm.html:97]: We should think about making this section into a partial as it has appeared in a few places already

Comment (petesfrench) [inline templates/download/server/arm.html:97]: We should think about making this section into a partial as it has appeared in a few places already

Comment (carkod) [reply templates/download/server/arm.html:97]: The layout totally changed, that's why I removed this current partial, but yes I plan to do it once I finish the next few pages, so I'm sure that they all look the same and have the same content, as sometimes UX may make some changes that changes things in a way I may need to refactor the partial.

File: templates/download/server/arm.html (hunk 1 above)
Comment (carkod) [reply templates/download/server/arm.html]: Fixed. It's also broken in prod.

File: templates/download/server/arm.html (hunk 3 above)
Comment (carkod) [reply templates/download/server/arm.html:97]: Removing this dup comment

**Comment by carkod** [comment]
@petesfrench all comments addressed except the partial, which I'll in the next few pages.

**Pull request merged**